flink-clients/src/main/java/org/apache/flink/client/program/MiniClusterClient.java

```diff
@@ -58,6 +58,11 @@ public MiniClusterClient(@Nonnull Configuration configuration, @Nonnull MiniClus
 		this.miniCluster = miniCluster;
 	}
 
+	@Override
+	public ClusterClient<MiniClusterId> duplicate() {
```

Comment:
Why is this client not properly duplicating?

Reply:
We can do that. It happens we can share the same instance of `MiniClusterClient` without worry about lifecycle management since they take no resource. But a new `MiniClusterClient` is more semantically proper.